### PR TITLE
Webpack installation

### PR DESCRIPTION
**Pull request by charlesx2013**
COMPLETE

FOR ALL THE HATERS OUT THERE (LOOKING AT YOU @warrenshen)

### CODE REVIEW

**Comment by chenhoward** [comment]
@warrenshen you don't like webpack?

**Comment by warrenshen** [comment]
@chenhoward I like webpack I just don't think @charlesx2013 needs it for this application.

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,9 +12,5 @@
 //
 //= require jquery
 //= require jquery_ujs
-
-//= require react
-//= require react_ujs
-//= require react_router
-//= require react_router_ujs
-//= require components
+r
```
Comment (warrenshen) [inline app/assets/javascripts/application.js]: `r`

File: app/assets/javascripts/authentication/app.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const App = ({ children }) => (
+  <div>
+    {children}
+  </div>
+)
+
+render((
+  <Router history={browserHistory}>
+    <Route path='/' name='StudentLoginPage' component={App} >
```
Comment (warrenshen) [inline app/assets/javascripts/authentication/app.js]: Why do you leave a space before ` >` here but not above?

File: app/assets/javascripts/authentication/app.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <IndexRoute component={StudentLoginPage} />
+      <Route path='sign_up' name='SignUpPage' component={SignUpPage}  />
```
Comment (warrenshen) [inline app/assets/javascripts/authentication/app.js]: Why two spaces `  />` but not above?

**Pull request merged**